### PR TITLE
Accessibility Pledge

### PR DESCRIPTION
**Pull request by mgifford**

### CODE REVIEW

File: _posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "CivicActions Accessibility Pledge"
+date:   2021-05-19
```
Comment (dmundra) [inline _posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md:4]: @mgifford 2021 typo.

**Review by dmundra — CHANGES_REQUESTED** [review]
@mgifford the filename for the new post has two names in it.

I was curious whether this post is also going to appear on the CA site then maybe we should link there instead of having two copies.

File: _posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md (hunk 1 above)
Comment (mgifford) [reply _posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md:4]: Or I have a time machine!!

**Comment by mgifford** [comment]
So this might eventually be something that lives on the main site, but it is just a bit harder to get it up there. I haven't even talked with Melinda about this yet.  

We wanted to post it somewhere that we had more control over first.

File: _posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md (hunk 1 above)
Comment (dmundra) [reply _posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md]: Yup. Thank you!

**Comment by dmundra** [comment]
> @dmundra So this might eventually be something that lives on the main site, but it is just a bit harder to get it up there. I haven't even talked with Melinda about this yet.
> 
> We wanted to post it somewhere that we had more control over first.

That is fair @mgifford. Looks like the filename is still broken. This is what it looks like:

`_posts/2021-02022-05-19-CivicActions-Accessibility-Pledge.md8-13-CivicActions-Creates-Open-Product-Accessibility-Template.md`

I presume it should be

`_posts/2022-05-19-CivicActions-Accessibility-Pledge.md`

**Comment by mgifford** [comment]
Yes, it should be

_posts/2022-05-19-CivicActions-Accessibility-Pledge.md

Let me go fix that.

**Review by dmundra — APPROVED** [review]

**Pull request merged**